src/components/attribute/EmptyState.tsx

```diff
@@ -0,0 +1,173 @@
+import { IDENTIFIER } from "@lib/extension";
+import React, { useEffect, useState } from "react";
+
+import { useClipboard } from "@lib/useClipboard";
+
+import { updatePullRequestLinkOnRecord } from "@lib/fields";
+import { withGitHubApi } from "@lib/github/api";
+import { getPrByUrl } from "@lib/github/getPr";
+import { LinkableRecord } from "@lib/linkableRecord";
+import { validPrUrl } from "@lib/validPrUrl";
+
+const LEARN_MORE_URL = "https://github.com/aha-develop/github";
+const ICON = aha.iconForExtensionIdentifier(IDENTIFIER);
+
+type MenuProps = {
+  record: LinkableRecord;
+  onPaste: Function;
+};
+
+const Menu = ({ record, onPaste }: MenuProps) => {
+  const handleSync = () => {
+    aha.command(`${IDENTIFIER}.sync`, { record });
+  };
+
+  return (
+    <aha-menu>
+      <aha-button slot="button" type="attribute" size="mini">
+        <aha-icon icon="fa-solid fa-ellipsis"></aha-icon>
+      </aha-button>
+      <aha-menu-item onClick={() => onPaste()}>Paste PR link</aha-menu-item>
```

Comment:
This is more UX feedback, this label is kind of confusing to me. Maybe "Manually link PR" or something? cc @trydionel

Reply:
Love that, yes. I struggled finding good wording for this.